content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -7,27 +7,22 @@ reviewers:
 - roycaihw
 - sttts
 content_type: task
+min-kubernetes-server-version: 1.16
```

Comment:
I agree. This statement implies that 1.16+ is required. Is there anything in this task that requires
v1.16 in order for the task to run?

Reply:
> Is there anything in this task that requires v1.16 in order for the task to run?

Yes: before v1.16 the CRD API was beta and the stable CRD API wasn't available.

Maybe we can tweak the version check shortcode to highlight when it makes sense to look at the documentation for a previous Kubernetes versioN?

Reply:
> 
> 
> > Is there anything in this task that requires v1.16 in order for the task to run?
> 
> Yes: before v1.16 the CRD API was beta and the stable CRD API wasn't available.
> 

So, if you remove all references to `v1beta1` in the text then the version requirement makes more sense.
Though, perhaps it also makes sense to add a note about the switch from v1beta1 to v1 for readers that
are running a previous version of Kubernetes (taken from the original text).
Investigating changes to the version shortcode is a separate issue.

Reply:
No. I don't agree. Please consider downstream distributions and existing deployments. I don't think all users are closely following the upstream changes. We should use the existing text "Make sure your Kubernetes cluster has a master version of 1.16.0 or higher to use apiextensions.k8s.io/v1, or 1.7.0 or higher for apiextensions.k8s.io/v1beta1". When 1.20 is out, we can safely remove the text about things before 1.16.

Reply:
@tengqm we provide copies of documentation for all supported releases.

I want to tell the reader that if they're on a previous release the can switch to a previous documentation version. If that's not clear enough, we can make the text about that bold or otherwise highlight it.

Reply:
> Extend the Kubernetes API with stable CustomResourceDefinitions

I wouldn't change the page title, but I can add an introduction that explains there used to be a beta version of the API and what that means for you.